### PR TITLE
Enable prefix usage with `cake bake all`

### PR DESCRIPTION
**Pull request by andtxr**

### CODE REVIEW

**Comment by andtxr** [comment]
Refers to #58

File: tests/TestCase/Shell/BakeShellTest.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ public function testAllWithModelName()
             ->with('Comments');
 
         $this->Shell->connection = '';
-        $this->Shell->params = [];
+        $this->Shell->params = ['prefix' => 'account'];
```
Comment (antograssiot) [inline tests/TestCase/Shell/BakeShellTest.php]: @andtxr The PR looks good but this doesn't look to really affect the actual test ? Am I missing something ?

Comment (andtxr) [inline tests/TestCase/Shell/BakeShellTest.php]: @antograssiot If the prefix param was passed before this PR, an exception would rise referring an invalid param. The new prefix option added to the parser covers it.

Comment (antograssiot) [inline tests/TestCase/Shell/BakeShellTest.php]: @andtxr well I'm pretty sure that I updated this line without the fix and it still works but nevermind

Comment (andtxr) [inline tests/TestCase/Shell/BakeShellTest.php]: So it is something to check again... This is not the behavior that I was expecting. Tonight I can check it better! Thanks @antograssiot   :+1:

**Comment by markstory** [comment]
Looks good. Was all that was needed was to capture and forward the parameter?

**Comment by antograssiot** [comment]
yep that's all

**Pull request merged**